src/actions/actionZindex.tsx

```diff
@@ -86,13 +87,13 @@ export const actionSendBackward: Action = {
   contextItemLabel: "labels.sendBackward",
   keyPriority: 40,
   commitToHistory: () => true,
-  keyTest: event => event[KEYS.META] && event.altKey && event.key === "B",
+  keyTest: event => event[KEYS.CTRL_OR_CMD] && event.key === "[",
```

Comment:
Does anyone know why this one is not working?

Reply:
Works for me. What browser are you using? Btw, is the `CMD + [` a standard here? I'm just asking, no idea if what we used before is or isn't.

Reply:
Chrome / Mac, all latest.. :/